src/components/VersionChooser/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Col, Form, Row } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import VersionSelect, { Version } from '../VersionSelect';
+import { gettext } from '../../utils';
+import styles from './styles.module.scss';
+
+type PublicProps = {
+  versions: Version[];
```

Comment:
The `Version` type (defined in the `VersionSelect` component) has a bad name but it is confusing because everything is called "version" now (the version number is `version`, a list of versions contains "version" objects), an add-on has versions, which have lots of data...